### ISSUE TITLE
Screen stays after start up.

### ISSUE THREAD

**Issue by RockyPilot** [rocksdanister/lively#70]
When I start up my computer after everything is finished I am left with a window sitting open. The title bar of the window says Lively MouseInput and the window is blank.

I can simply click the X and the box closes. But it's annoying having this box just sitting there every time I boot my computer. 

I am running windows 10 Version 1909  OS build: 18363.836
Intel Core i9-9900K CPUa@ 3.6GHz (not overclocked)
16GB of fast RAM (not overclocked)

Wallpaper is: Fractal Clock

![Capture](https://user-images.githubusercontent.com/66171686/83330781-339cf400-a257-11ea-86aa-098a94673db7.JPG)

**Comment by rocksdanister** [comment]
What's that widget like thing I see on your window titlebar?

This window is not suppose to appear, its for detecting mouse clicks for the wallpapers.

> I can simply click the X and the box closes. But it's annoying having this box just sitting there every time I boot my computer.

In settings-> interaction if you set it to Off won't see this... but you will lose mouse input in wallpapers.

**Comment by RockyPilot** [comment]
That fixed the problem. It was the Nvidia Desktop Manager. I am guessing it installed as part of the software for my RTX4000 video card. I had been trying to find a way to disable it. After you asked about it I was able to finally find it and disable it. After rebooting my computer the window doesn't pop up anymore. Thank you for the help my issue has been resolved.  Also, thanks for the program. It's very cool. I wish MicroSoft would just build stuff like this into it's operating system.
